tests/integ/test_function.py

```diff
@@ -538,9 +570,14 @@ def test_basic_numerical_operations_negative(session):
     assert "'CEIL' expected Column or str, got: <class 'list'>" in str(ex_info)
 
 
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="BUG: substring raises unsupported operand type(s) for -: 'str' and 'int'",
+)
 def test_basic_string_operations(session):
     # Substring
-    df = session.sql("select 'a not that long string'").to_df("a")
+    df = session.create_dataframe(["a not that long string"], schema=["a"])
+    # df = session.sql("select 'a not that long string'").to_df("a")
```

Comment:
Shall we remove this line? Also, is this bug fix assigned?

Reply:
yes, line removed and jira number added